components/authz-service/engine/opa/opa_internal_test.go

```diff
@@ -549,3 +429,866 @@ func randomPolicies(i int) []map[string]interface{} {
 
 	return newPolicies
 }
+
+// v2 benchmarks
+
+var (
+	allProjects = []string{
```

Comment:
most tests use this default set of projects, except for the tests that increase total projects.